build.gradle

```diff
@@ -63,6 +63,7 @@ allprojects {
     apply plugin: "signing"
     apply plugin: 'com.github.ben-manes.versions'
     apply plugin: "jacoco"
+    apply plugin: "maven-publish"
```

Comment:
What does this do?

Reply:
That was my lame attempt to publish locally using "gradle publishToMavenLocal" task until I reached out.  I will remove it.